junit5-engine/src/main/java/org/junit/gen5/engine/junit5/descriptor/JUnit5TestableFactory.java

```diff
@@ -155,7 +155,11 @@ private Method findMethod(Class<?> clazz, String methodName, Class<?>[] paramete
 	}
 
 	private Class<?> loadClassByName(String className) {
-		return ReflectionUtils.loadClass(className).orElse(null);
+		return ReflectionUtils.loadClass(className).orElseThrow(() -> createCannotLoadClassException(className));
+	}
```

Comment:
Hi @kcooney,

Any chance you'll get around to creating that issue/PR this week?

If not, I think we'll go ahead and push the proposed change anyway.

cheers

Sam